### DISCUSSION TITLE
Avian ESC programming support along side DSM programming

### DISCUSSION THREAD

**Discussion by BuddhaSource** [pascallanger/DIY-Multiprotocol-TX-Module#826] [Ideas]
Specktrum has made it easy to program reverse thrust on their esc using a transmitter. Like Lua Script for DSM programming, this will also be an excellent option for EdgeTx transmitters.

Here is the link to how this is currently done https://www.youtube.com/watch?v=5SdE2Ok2TRU

**Comment by frankiearzu** [comment]
AVIAN ESC uses what is called "TextGen".. 
It needs to update the EdgeTX firmare to 2.9.0-nightly to work (in development).

I have a few versioms o EdgeTX 2.8.1 + TextGen zip files on my site for TX16S and Zorro.. If you have another radio, let me know and i can compile the firmware for that radio.
https://github.com/frankiearzu/DSMTools
